app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -264,6 +264,12 @@ def raw_connect_v4(options = {})
       # Get the timeout from the configuration:
       timeout, = ems_timeouts(:ems_redhat, options[:service])
 
+      # The constructor of the SDK expects a list of certificates, but that list can't be empty, or contain only 'nil'
+      # values, so we need to check the value passed and make a list only if it won't be empty. If it will be empty then
+      # we should just pass 'nil'.
+      ca_certs = options[:ca_certs]
+      ca_certs = [ca_certs] if ca_certs
```

Comment:
No problem. The content of that database is one string contaning one ore more certificates, in PEM format. For example:

```
-----BEGIN CERTIFICATE----
The first certificate.
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
The second certificate.
-----END CERTIFICATE-----
...
```

For user convenience, the oVirt SDK accepts them in separate strings, which would require us to split them here, but not using command but the `BEGIN/END` marks. Fortunately, also for user conveninence, the oVirt SDK also accepts multiple certificates in the same string, so we don't need to split anything here. The only thing that the SDK doesn't accept is `[nil]` (well, it accepts it, but it doesn't have the meaning that we need: it trusts no CA instead of trusting any CA), so that is the only case that we need to check here.